sorc/ncep_post.fd/BNDLYR.f

```diff
@@ -294,9 +308,14 @@ SUBROUTINE BNDLYR(PBND,TBND,QBND,RHBND,UBND,VBND,       &
           DO J=JSTA_M,JEND_M
             DO I=2,IM-1
               IF(PVSUM(I,J,LBND)/=0.)THEN
+              IF(UH(I,J,LBND)<spval.and.VH(I,J,LBND)<spval)THEN
                 RPVSUM         = 1./PVSUM(I,J,LBND)
                 UBND(I,J,LBND) = UBND(I,J,LBND)*RPVSUM
                 VBND(I,J,LBND) = VBND(I,J,LBND)*RPVSUM
```

Comment:
line 311, please add variable pvsum,ubnd and vbnd.
@JesseMeng-NOAA

Reply:
will do

Reply:
done